packages/core/src/domain/configuration.ts

```diff
@@ -174,12 +184,15 @@ export function buildConfiguration(userConfiguration: UserConfiguration, buildEn
       configuration.replica = {
         applicationId: userConfiguration.replica.applicationId,
         internalMonitoringEndpoint: getEndpoint(
-          'browser',
+          EndpointType.LOGS,
           replicaTransportConfiguration,
           'browser-agent-internal-monitoring'
         ),
-        logsEndpoint: getEndpoint('browser', replicaTransportConfiguration),
-        rumEndpoint: getEndpoint('rum', replicaTransportConfiguration),
+        logsEndpoint: getEndpoint(EndpointType.LOGS, replicaTransportConfiguration),
+        rumEndpoint: getEndpoint(EndpointType.RUM, replicaTransportConfiguration),
+      }
+      if (transportConfiguration.site !== INTAKE_SITE[Datacenter.US]) {
```

Comment:
Do we need this check?

Reply:
if the site is `INTAKE_SITE[Datacenter.US]` we will end up pushing the same URLs twice. That's why I added this condition.

Reply:
since it is not planned to have replica on a prod configuration and if that happens the impact should be negligible, I'd be in favor of removing this check for simplicity sake.